.circleci/config.yml

```diff
@@ -48,9 +48,47 @@ jobs:
             sudo mv /tmp/kubebuilder_2.0.0-alpha.4_${os}_${arch} /usr/local/kubebuilder
             export PATH=$PATH:/usr/local/kubebuilder/bin
       - run: make test
+
+  release:
+    docker:
+      - image: circleci/golang:1.12
+        environment:
+          - GO111MODULE=on
+    working_directory: /go/src/github.com/ory/oathkeeper-k8s-controller
+    steps:
+      - checkout
+      - run:
+          command: |
+            echo 'export DOCKER_FULL_TAG=$(echo $CIRCLE_TAG | tr '+' '_')' >> $BASH_ENV
+            echo 'export DOCKER_SHORT_TAG=$(echo $CIRCLE_TAG | cut -d '+' -f1)' >> $BASH_ENV
+            ./.circleci/release_name.bash
+            source $BASH_ENV
+      - setup_remote_docker
+      - run: docker login --username "$DOCKER_USERNAME" --password "$DOCKER_PASSWORD"
+      - run: curl -sL https://git.io/goreleaser | bash
```

Comment:
Goreleaser must be configured: https://goreleaser.com/

Alternatively just `docker build` and `docker push`

ps: The current docker file does not work with goreleaser, see: https://goreleaser.com/customization/#Docker

Reply:
I have pushed goreleaser yml config file, based on oathkeeper, however, I think that few things needs to be done on your side such as configuring GITHUB_TOKEN etc. Regarding dockerfile I tool a look at docu and compared it with oathkeeper dockcerfile and didn't notice any  missings. Could you point me ?:)

Reply:
Go releaser builds docker images in such a way where it first builds the binaries for all platform, then builds a docker image which simply copies the linux binary into the container. So there is no actual build step involved.